endoreg_db/models/media/video/video_file.py

```
@@ -80,10 +80,16 @@
         VideoState,
         ModelMeta,
         VideoImportMeta,
-    )
+    )   
+class VideoQuerySet(models.QuerySet):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)